### DISCUSSION TITLE
How can I "install" pocketbase several times to have my separate projects?

### DISCUSSION THREAD

**Discussion by jossmanbr2022** [pocketbase/pocketbase#281] [Q&A]
I would like to have my projects separated to identify and export them more easily, I don't know if this is possible... I appreciate the help

**Comment by jimafisk** [comment]
This sounds similar to https://github.com/pocketbase/pocketbase/discussions/97

**Comment by ganigeorgiev** [comment]
You could start the `pocketbase` executable multiple times with different `--http/--https` and `--dir` flags, eg:
```sh
./pocketbase serve --dir="./pb_data1" --http="127.0.0.1:8091" & \
./pocketbase serve --dir="./pb_data2" --http="127.0.0.1:8092" & \
./pocketbase serve --dir="./pb_data3" --http="127.0.0.1:8093" & \
./pocketbase serve --dir="./pb_data4" --http="127.0.0.1:8094" &
```

To terminate all processes, you could run something like: `sudo killall pocketbase`
